lib/school_house_web/templates/post/post.html.heex

```diff
@@ -19,7 +19,9 @@
     </div>
 
     <div class="py-2 text-center">
-      <h2 class="text-medium leading-8 tracking-tight text-primary dark:text-primary-dark sm:text-large "><%= gettext("By") %> <%= @post.author %></h2>
+      <h2 class="text-medium leading-8 tracking-tight text-primary dark:text-primary-dark sm:text-large ">
+        <%= gettext("By") %> <%= @post.author %> | <%= gettext("Posted") <%= @post.date %>
```

Comment:
I think the CI errors were happening because we missed a closing `%>` here (based on CI logs) 🙂 :

```suggestion
        <%= gettext("By") %> <%= @post.author %> | <%= gettext("Posted") %> <%= @post.date %>
```

Reply:
Ahh, good catch!